src/lib/compiler/compileData.js

```diff
@@ -43,7 +43,7 @@ import { assetFilename } from "../helpers/gbstudio";
 const indexById = indexBy("id");
 
 const DATA_PTRS_BANK = 5;
-const NUM_MUSIC_BANKS = 8;
+const NUM_MUSIC_BANKS = 1;//Auto expands, needed to calculate first music bank.
```

Comment:
since this is always 1 could we get rid of it and also the `for (let i = 0; i < NUM_MUSIC_BANKS; i++) {` loop in line 351 to be removed?